engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/UnresolvedConversion.java

```diff
@@ -52,7 +52,7 @@ public Function resolveFor(Type into, Type from) {
         current = current.getSupertype();
       }
     }
-    return null;
+    return scope.lookupConversionDefinition(ctx.getBuiltins().any(), into);
```

Comment:
Isn't this going to eliminate the need for all those `....Any.Any` imports introduced by @Akirathan ?

Reply:
Nope. `Comparable.from (_:Any) = Default_Comparator` is defined in `Ordering.enso`. So the minimal import for, say, this snippet `5 + 3 == 8` should be `from Standard.Base.Data.Ordering import all`.

Reply:
I was wondering: _"how will I find these `....Any.Any` imports?"_

> So the minimal import ... should be `from Standard.Base.Data.Ordering import all`.

Now I no longer need to solve that question....